### PR TITLE
add CONTRIBUTING.md

### LINKED ISSUE #13: Create CONTRIBUTING doc
[working template](https://github.com/atom/atom/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by capsulecorplab**
closes #13

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Submitting a pull request (PR)
+
+Note: we will be using the [ZenHub for GitHub](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd?hl=en-US) chrome extension.
+
+1. Submit a PR, by setting your feature branch as `HEAD` and `develop` branch as `root`. 
+2. Connect PR to its corresponding issue that it closes. 
+3. Set ZenHub pipeline to "Review/QA"
+4. Assign a reviewer.
+5. When ready, add the `author ready` label to indicate that you are satisfied with the PR to be merged.
+
```
Comment (itsevalieu) [inline CONTRIBUTING.md:56]: What's our decision on deleting branches when an issue is resolved?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+```
+git checkout develop
+git checkout -b issue42-add-logarithms
+```
+
+Use a present tense commit message, preferably one that's no more than 50 characters in length:
+
+```
+git commit -am "add logarithms"
+```
```
Comment (itsevalieu) [inline CONTRIBUTING.md:32]: Are we starting commit messages with capitalization or does that not matter? ie: "Add logarithms" vs "add logarithms."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+git checkout develop
+git checkout -b issue42-add-logarithms
+```
+
+Use a present tense commit message, preferably one that's no more than 50 characters in length:
+
+```
+git commit -am "add logarithms"
```
Comment (itsevalieu) [inline CONTRIBUTING.md]: Are we starting commit messages with capitalization or does it not matter? ie: "Add logarithms" vs "add logarithms."

File: CONTRIBUTING.md (hunk 1 above)
Comment (capsulecorplab) [reply CONTRIBUTING.md:56]: I generally delete branches after they're merged, as it keeps the workspace looking clean. We can always recreate branches by checking commit hashes.

File: CONTRIBUTING.md (hunk 2 above)
Comment (capsulecorplab) [reply CONTRIBUTING.md:32]: I'm thinking we should reserve capitalized commit messages for "_important_" commits - emphasis on the subjectivity of the word "_important_".

Comment (itsevalieu) [reply CONTRIBUTING.md:32]: Sounds goood! Maybe should note that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (itsevalieu) [reply CONTRIBUTING.md:56]: That sounds fine to me, so where in the doc can we put that note, probably should put the git action for it too

File: CONTRIBUTING.md (hunk 2 above)
Comment (capsulecorplab) [reply CONTRIBUTING.md:32]: added some notes

File: CONTRIBUTING.md (hunk 1 above)
Comment (capsulecorplab) [reply CONTRIBUTING.md:56]: Added an extra step. I figured people will just use the "delete branch" button on GitHub in the PR, you can also do it via the "branches" tab.

**Comment by capsulecorplab** [comment]
Changed `develop` to `dev`

**Pull request merged**